### PR TITLE
D8CORE-2644: Tests for embeddable media form alters

### PR DESCRIPTION
**Pull request by imonroe**
# READY FOR REVIEW

# Summary
- Acceptance tests to go with https://github.com/SU-SWS/stanford_profile_helper/pull/27
- These tests won't pass until that PR is merged.

# Needed By (Date)
- next code cutoff.

# Urgency
- 3 out of 10.

# Steps to Test

1. Let the tests run, make sure there's no failures.

# Associated Issues and/or People
- https://github.com/SU-SWS/stanford_profile_helper/pull/27

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: tests/codeception/acceptance/Media/MediaFormsCest.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    $I->seeLink('request support', 'https://stanford.service-now.com/it_services?id=sc_cat_item&sys_id=83daed294f4143009a9a97411310c70a');
+    $I->click('Delete');
+    $I->seeInCurrentUrl('delete');
+    $I->click('Delete');
+    $I->dontSeeLink('Test embed');
+    $I->seeInCurrentUrl('/admin/content/media');
+  }
+
+
+
```
Comment (pookmish) [inline tests/codeception/acceptance/Media/MediaFormsCest.php:30]: Too many empty lines.
also make sure there is one empty line at the bottom of the file.

File: tests/codeception/acceptance/Media/MediaFormsCest.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ */
+class MediaFormsCest {
+
+  /**
+   * Test embeddables form alters
+   */
+  public function testFormAlters(AcceptanceTester $I) {
+    $I->logInWithRole('administrator');
+    $I->amOnPage('/media/add/embeddable');
+    $I->seeLink('request support', 'https://stanford.service-now.com/it_services?id=sc_cat_item&sys_id=83daed294f4143009a9a97411310c70a');
```
Comment (pookmish) [inline tests/codeception/acceptance/Media/MediaFormsCest.php:14]: This link won't be visible for administrators. You should use composer to target the `stanford_profile_helper` branch related to these tests to verify they pass.

**Review by pookmish — CHANGES_REQUESTED** [review]

**Comment by pookmish** [comment]
@imonroe any progress on this?

File: tests/codeception/acceptance/Media/MediaFormsCest.php (hunk 1 above)
Comment (imonroe) [reply tests/codeception/acceptance/Media/MediaFormsCest.php]: fixed.

File: tests/codeception/acceptance/Media/MediaFormsCest.php (hunk 2 above)
Comment (imonroe) [reply tests/codeception/acceptance/Media/MediaFormsCest.php]: fixed.

**Review by pookmish — APPROVED** [review]
g2g after the tests pass

**Review by sherakama — APPROVED** [review]
GTG

**Pull request merged**